src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -8,89 +8,96 @@
-            az login --identity -u /subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID
-    """
+type: command
+short-summary: Log in to Azure.
+examples:
+    - name: Log in interactively.
+      text: >
+        az login
+    - name: Log in with user name and password. This doesn't work with Microsoft accounts or accounts that have two-factor authentication enabled. Use -p=secret if the first character of the password is '-'.
+      text: >
+        az login -u johndoe@contoso.com -p VerySecret
+    - name: Log in with a service principal using client secret. Use -p=secret if the first character of the password is '-'.
+      text: >
+        az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p VerySecret --tenant contoso.onmicrosoft.com
+    - name: Log in with a service principal using client certificate.
+      text: >
+        az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p ~/mycertfile.pem --tenant contoso.onmicrosoft.com
+    - name: Log in using a VM's system assigned identity
+      text: >
+        az login --identity
+    - name: Log in using a VM's user assigned identity. Client or object ids of the service identity also work
+      text: >
+        az login --identity -u /subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID
+"""
 
 helps['account'] = """
-    type: group
-    short-summary: Manage Azure subscription information.
+type: group
+short-summary: Manage Azure subscription information.
 """
 
 helps['account clear'] = """
-    type: command
-    short-summary: Clear all subscriptions from the CLI's local cache.
-    long-summary: To clear the current subscription, use 'az logout'.
+type: command
+short-summary: Clear all subscriptions from the CLI's local cache.
+long-summary: To clear the current subscription, use 'az logout'.
 """
 
 helps['account list'] = """
-    type: command
-    short-summary: Get a list of subscriptions for the logged in account.
+type: command
+short-summary: Get a list of subscriptions for the logged in account.
 """
 
 helps['account list-locations'] = """
-    type: command
-    short-summary: List supported regions for the current subscription.
+type: command
+short-summary: List supported regions for the current subscription.
 """
 
 helps['account show'] = """
-    type: command
-    short-summary: Get the details of a subscription.
-    long-summary: If no subscription is specified, shows the current subscription.
+type: command
+short-summary: Get the details of a subscription.
+long-summary: If no subscription is specified, shows the current subscription.
 """
 
 helps['account set'] = """
-    type: command
-    short-summary: Set a subscription to be the current active subscription.
+type: command
+short-summary: Set a subscription to be the current active subscription.
 """
 
 helps['account show'] = """
```

Comment:
⚠ `--sdk-auth` should be deprecated for `account show` because it is regurgitating the credential which is not a secure practice.